lib/components/narrative/line-itin/itin-summary.tsx

```diff
@@ -0,0 +1,247 @@
+import { connect } from 'react-redux'
+import { FormattedMessage, FormattedNumber } from 'react-intl'
+// TYPESCRIPT TODO: add leg, itinerary types all over the place here
```

Comment:
You can import them from `@opentripplanner/types` and get rid of `PropTypes`.

Reply:
Feels so good. Done in 6d5bc90e